openstates/in/bills.py

```diff
@@ -77,9 +81,127 @@ def get_name(self,random_json):
+
+            motion = " ".join(lines[4].split()[:-2])
+            try:
+                yeas = int(lines[4].split()[-1])
+                nays = int(lines[5].split()[-1])
+                excused = int(lines[6].split()[-1])
+                not_voting = int(lines[7].split()[-1])
+            except ValueError:
+                self.logger.warning("Vote format is weird, skipping")
+                continue
+            other_count = excused + not_voting
+
+            vote = Vote(chamber,vote_date,motion,passed,yeas,nays,other_count,yes_votes=[],no_votes=[],other_votes=[])
+
+            vote.add_source(proxy_link)
+
+            currently_counting = ""
+
+            possible_vote_lines = lines[8:]
+            for l in possible_vote_lines:
+                l = l.replace("\xc2\xa0"," -")
+                if "yea -" in l.lower():
+                    currently_counting = "yes_votes"
+                elif "nay -" in l.lower():
+                    currently_counting = "no_votes"
+                elif "excused -" in l.lower():
+                    currently_counting = "other_votes"
+                elif "not voting -" in l.lower():
+                    currently_counting = "other_votes"
+                elif currently_counting == "":
+                    pass
+                elif re.search(r'v\. \d\.\d',l):
+                    #this gets rid of the version number
+                    #which is often found at the bottom of the doc
+                    pass
+                else:
+                    voters = l.split("  ")
+                    for v in voters:
+                        if v.strip():
+                            vote[currently_counting].append(v.strip())
+
+            assert (len(vote["yes_votes"]) == vote["yes_count"],
+                "Yes vote counts ({count}) don't match count of actual votes ({actual})".format(count=vote["yes_count"],actual=len(vote["yes_votes"])))
+            assert (len(vote["no_votes"]) == vote["no_count"],
+                "No vote counts ({count}) don't match count of actual votes ({actual})".format(count=vote["no_count"],actual=len(vote["no_votes"])))
+            assert (len(vote["other_votes"]) == vote["other_count"],
+                "Other vote counts ({count}) don't match count of actual votes ({actual})".format(count=vote["other_count"],actual=len(vote["other_votes"])))
+            
+
+            #vote should have the majority if it passed
+            if vote["passed"] and vote["yes_count"] <= vote["no_count"]:
+                raise AssertionError("The vote should have at least a majority if it passed.")
+
+            #if vote has the majority and didn't pass, throw a warning
+            #this is possible in supermajority cases,
+            #but we might want to think about it if it starts happening lots
+            if not vote["passed"] and vote["yes_count"] > vote["no_count"]:
+                self.logger.warning("Vote got a majority but didn't pass. Probably worth a look.")
```

Comment:
As long as we're doing these checks, can we do a hard `assert` to make sure that if a bill _does_ have supermajority that it absolutely _does_ pass?

From poking around I don't think that Indiana _has_ anything that requires a supermajority, even veto overrides or constitutional amendments, but I could be wrong (`http://www.in.gov/ipas/2425.htm`).